packages/merge-on-green/src/merge-logic.ts

```diff
@@ -640,7 +640,7 @@ mergeOnGreen.removeLabel = async function removeLabel(
   github: GitHubAPI
 ) {
   try {
-    github.issues.removeLabel({
+    await github.issues.removeLabel({
```

Comment:
@sofisl look what I found!  A bug was hiding here!

Reply:
Beautiful :) Thank you!!